Sources/Fullscreen/Consent/Settings/ConsentViewCell.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+import UIKit
+
+public struct ConsentViewCellModel {
```

Comment:
Could you move this to a different file? I think it's better if we make files (there's probably some exceptions) for most classes and structs (protocols don't need their own files, but again, with exceptions).

Reply:
I can make a models file